concourse/pipelines/container-build.jsonnet

```diff
@@ -215,8 +276,32 @@ local BuildContainerImage(image) = buildcontainerimgjob {
                 args: [
                   '-exc',
                   'for f in darwin linux windows; do' +
-                  '  gsutil cp $f/daisy gs://compute-image-tools/latest/$f/daisy;' +
-                  '  gsutil acl ch -u AllUsers:R gs://compute-image-tools/latest/$f/daisy;' +
+                  '  for t in latest release; do' +
+                  '    gsutil cp $f/daisy gs://compute-image-tools/$t/$f/daisy;' +
+                  '    gsutil acl ch -u AllUsers:R gs://compute-image-tools/$t/$f/daisy;' +
+                  '  done;' +
+                  'done',
+                ],
+              },
+            },
+          },
+          //  Add release and latest tags to the staged container.
+          {
+            task: 'promote-container',
+            config: {
+              platform: 'linux',
+              image_resource: {
+                type: 'registry-image',
+                source: { repository: 'google/cloud-sdk', tag: 'alpine' },
+              },
+              run: {
+                path: 'sh',
+                args: [
+                  '-exc',
+                  'for t in latest release; do' +
+                  '  gcloud container images add-tag --quiet' +
+                  '    gcr.io/compute-image-tools/daisy:((.:compute-daisy-commit-sha))' +
+                  '    gcr.io/compute-image-tools/daisy:$t;' +
```

Comment:
so it's your intention to keep both tags up to date, here? so each new image version gets 3?

Reply:
Good question -- my intent was to keep the same behavior as the [current cloudbuild results](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools_cloudbuild.yaml). If I were creating this new, I'd propose two tags (the git sha and `latest`), but since existing people might be using `release`, I'd propose we keep publishing to that tag.